### ISSUE TITLE
Use ESVN_USER and ESVN_PASSWORD as subversion.eclass would do

### ISSUE THREAD

**Issue by mkolleck** [mgorny/smart-live-rebuild#17]
A live ebuild needing to authenticate to the Subversion server can use ESVN_USER and ESVN_PASSWORD from the environment. However, when later updating using smart-live-rebuild, the user is asked for authentication information even though both environment variables are set.

I have the following patch in /etc/portage/patches for now to work around this issue:

```
--- a/smartliverebuild/vcs/subversion.py
+++ b/smartliverebuild/vcs/subversion.py
@@ -8,7 +8,7 @@
 
 class SubversionSupport(RemoteVCSSupport):
        reqenv = ['ESVN_REPO_URI', 'ESVN_STORE_DIR', 'ESVN_WC_REVISION']
-       optenv = ['ESVN_REVISION']
+       optenv = ['ESVN_REVISION', 'ESVN_USER', 'ESVN_PASSWORD']
 
        revre = re.compile('(?m)^Last Changed Rev: (\d+)$')
 
@@ -43,5 +43,8 @@
        @property
        def updatecmd(self):
                # XXX: branch?
+               if self.env['ESVN_USER'] and self.env['ESVN_PASSWORD']:
+                       return 'svn --config-dir %s/.subversion info --username=%s --password="%s" %s' % (
+                               self.env['ESVN_STORE_DIR'], self.env['ESVN_USER'], self.env['ESVN_PASSWORD'], self.env['ESVN_REPO_URI'])
                return 'svn --config-dir %s/.subversion info %s' % (
                        self.env['ESVN_STORE_DIR'], self.env['ESVN_REPO_URI'])
```

**Comment by mgorny** [comment]
Could you make a pull request for this? However, please inject extra args into the basic template rather than creating two templates.

**Comment by mgorny** [comment]
Fixed via #18
